lib/ui/blocs/deeplink/deeplink_bloc.dart

```diff
@@ -37,10 +79,7 @@ class DeeplinkBloc extends Bloc<DeeplinkEvent, DeeplinkState> {
   Future<void> _incomingFirebaseDeepLink(_IncomingFirebaseDeepLink event, Emitter<DeeplinkState> emit) async {
     /// fetch data from link.
     final queryParams = event.link.queryParameters;
-    if (queryParams.isNotEmpty &&
-        queryParams.containsKey('code') &&
-        queryParams.containsKey('chain') &&
-        queryParams.containsKey('dao')) {
+    if (queryParams.isNotEmpty && queryParams.containsKey('code') && queryParams.containsKey('chain')) {

```

Comment:
FYI "dao" is actually not mandatory. This could be in another PR. Since it has nothing to do with this one.